### DISCUSSION TITLE
Empty location data

### DISCUSSION THREAD

**Discussion by relu91** [plausible/analytics#3550] [Self-Hosted Support]
Hello everybody, first of all, I want to thank the Plausible team for making this great tool and giving us the ability to self-host it. I have a question regarding location data. In our self-hosted instance, I'm not seeing any Location data:
![image](https://github.com/plausible/analytics/assets/3170227/3e3ffe40-41a4-447c-bf8f-32d6bcdd6f8e)

To be fair I didn't set up [MaxMind](https://www.maxmind.com/) services, but from the [docs](https://plausible.io/docs/self-hosting-configuration#ip-geolocation) I understand that this is an optional step and the IP geolocation should work nonetheless (but less precise). Is that so?

**Comment by ruslandoga** [comment]
👋 @relu91 

Yes, the IP geolocation should work without any extra setup. What you are seeing is probably a problem with the proxy and the headers it sends.

**Reply by ruslandoga** [reply]
[This comment](https://github.com/plausible/analytics/discussions/3248#discussioncomment-6670217) from a similar discussion might be useful.

**Reply by relu91** [reply]
Thank you for the quick response! Indeed is something with the proxy. I used the snippet above and the x-forwarded-for points to a local IP. I have to figure out how to tell Traefik to forward the right IP.

**Reply by jordanlambrecht** [reply]
@relu91 Did you figure this out? Same problem.

**Reply by relu91** [reply]
I think the todo item was flagged as not a critical priority, and I never followed up. Sadly, I don't have access to that platform anymore, so I can't help you here. Sorry.
